### ISSUE TITLE
Fork icons seem to be broken on macOS

### ISSUE THREAD

**Issue by iremenerys** [ForkIssues/Tracker#1213]
I'm using Fork Version 2.1.2 (which is the latest version apparently) on macOS High Sierra 10.13.6, 

My icons started to look strange starting today. See the screenshot I've attached below:

![image](https://user-images.githubusercontent.com/8233171/101363885-f8891680-38b2-11eb-8187-477716ca0237.png)

**Comment by axiac** [comment]
The same for Fork 2.1.1. I run it on macOS 10.13 (High Sierra) and I guess the bug has been introduced in 2.1 (it worked fine in 2.0) with the changes for BigSur.

**Comment by DanPristupov** [comment]
> I guess the bug has been introduced in 2.1 (it worked fine in 2.0) with the changes for BigSur.

We didn't touch those icons :(. I think the new XCode 12 breaks some images on old MacOS versions.

I made an attempt to fix this. Could you check if this version draws the icons correctly?

https://cdn.fork.dev/prerelease/Fork-2.1.10.dmg

**Comment by axiac** [comment]
Thank you for your answer.
Unfortunately it shows the icons the same way, too large and truncated.

<img width="461" alt="image" src="https://user-images.githubusercontent.com/1086598/101464455-56673e00-3947-11eb-8afc-32189ec4b595.png">

**Comment by axiac** [comment]
I just noticed that in both screen captures the icons for remotes are displayed correctly (and they are different icons).
Maybe this detail can help you identify and solve the problem.

**Comment by DanPristupov** [comment]
Can you try this one please? https://cdn.fork.dev/prerelease/Fork-2.1.11.dmg

> I just noticed that in both screen captures the icons for remotes are displayed correctly (and they are different icons).

I will check this, thanks.

**Comment by axiac** [comment]
Everything went back to normal in 2.1.11.
Thank you!

<img width="379" alt="image" src="https://user-images.githubusercontent.com/1086598/101467713-58330080-394b-11eb-9e1b-d0fdff612602.png">

**Comment by DanPristupov** [comment]
☀️😊

OK. We will make this fix official in 2.2.

Please let me know if you notice any other broken icon.

**Comment by Feuermurmel** [comment]
2.1.11 also fixes this for me. I noticed something else related to icon rendering:

![image](https://user-images.githubusercontent.com/50332/101624863-e32efc00-3a1a-11eb-821d-5804b4884fc2.png)

The rendering of the icon used for remotes seems off. This is on a non-retina display on macOS 10.13.6.

I did not notice this before and can't say whether or when exactly this changed. On that note, is there an archive of old releases for Fork?
